cmd/otelcontribcol/go.mod

```diff
@@ -0,0 +1,1391 @@
+// Code generated by "go.opentelemetry.io/collector/cmd/builder". DO NOT EDIT.
```

Comment:
this file should not be in the source tree.

Reply:
Thank you for catching that, I've updated that 👍